src/stream/src/from_proto/source.rs

```diff
@@ -48,7 +50,13 @@ impl ExecutorBuilder for SourceExecutorBuilder {
             node.properties.clone(),
             node.get_info()?.clone(),
             params.env.source_manager_ref(),
-            params.env.connector_source_endpoint(),
+            params
```

Comment:
You have to pass the `Option` all the way down to the specific sources that really needs to unwrap and take the connector endpoint out. Not all sources need connecting to connector node. This fails the CI now. 

Actually you can pass the whole connector param in the parameter instead of the endpoint, as what we do for the sink.